src/server/handlers/launchpad.js

```diff
@@ -210,6 +218,47 @@ const getSnapcraftYaml = (owner, name, token) => {
     });
 };
 
+const verifySnapNameRegistered = (name) => {
+  return fetch(`${conf.get('STORE_MACAROON_API_URL')}/acl/`, {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
```

Comment:
I can't remember if `fetch` does this by itself, but we should make sure to pass `Accept: application/json` as well, even if the store doesn't enforce that today.